pkgdown/_pkgdown.yml

```diff
@@ -42,26 +54,30 @@ reference:
   - dm_get_colors
   - dm_get_available_colors
 
+- title: Zooming to a table of a `dm`
+  contents:
+  - dm_zoom_to_tbl
+  - dm_insert_zoomed_tbl
+  - dm_update_zoomed_tbl
+  - dm_zoom_out
+
 - title: Flattening
   contents:
   - dm_flatten_to_tbl
+  - dm_squash_to_tbl
   - dm_join_to_tbl
 
 - title: Filtering
   contents:
   - dm_filter
   - dm_apply_filters
+  - dm_apply_filters_to_tbl
 
 - title: Interaction with DBs
   contents:
   - dm_copy_to
   - dm_learn_from_db
 
-- title: Utilizing foreign key relations
-  contents:
-  - dm_is_referenced
-  - dm_get_referencing_tables
-
 - title: Table surgery
   contents:
   - decompose_table
```

Comment:
Could we do `dm_separate_tbl()` and `dm_unite_tbl()` instead, without corresponding non-table functions?

Reply:
sounds good! The `dm_unite_tbl()` would then be pretty much like the `dm_flatten_to_tbl()` with the result reinserted into the `dm`.

Is this still material for milestone 0.1.0.?

Reply:
see #211